tests/filters/test_monster_filter.py

```diff
@@ -201,6 +201,21 @@ def test_form(self):
         self.assertFalse(mon_filter.check_event(create_event({'form': 2})))
         self.assertFalse(mon_filter.check_event(create_event({'form': 999})))
 
+    def test_costume(self):
```

Comment:
Both this and the forms should be updated to use name

Reply:
Tried this and failed, leaving as ID for now.